Firestore/Source/Local/FSTLRUGarbageCollector.h

```diff
@@ -29,22 +29,39 @@
 
 extern const firebase::firestore::model::ListenSequenceNumber kFSTListenSequenceNumberInvalid;
 
-struct FSTLruGcParams {
-  static FSTLruGcParams Default() {
-    return FSTLruGcParams{.minBytesThreshold = 100 * 1024 * 1024,
-                          .percentileToCollect = 10,
-                          .maximumSequenceNumbersToCollect = 1000};
+namespace firebase {
+namespace firestore {
+namespace local {
+
+struct LruParams {
+  static LruParams Default() {
+    return LruParams{100 * 1024 * 1024, 10, 1000};
   }
 
-  static FSTLruGcParams Disabled() {
-    return FSTLruGcParams{.minBytesThreshold = kFIRFirestorePersistenceCacheSizeUnlimited};
+  static LruParams Disabled() {
+    return LruParams{kFIRFirestorePersistenceCacheSizeUnlimited, 0, 0};
   }
 
   long minBytesThreshold;
   int percentileToCollect;
   int maximumSequenceNumbersToCollect;
 };
 
+struct LruResults {
+  static LruResults DidNotRun() {
+    return LruResults{NO, 0, 0, 0};
+  }
+
+  BOOL didRun;
```

Comment:
C++ has a boolean type `bool` whose literals are `true` and `false`. (`BOOL`, `YES`, `NO` are Objective-C).